sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricsExporter.cs

```diff
@@ -24,14 +33,66 @@ public LiveMetricsExporter(DoubleBuffer doubleBuffer, LiveMetricsExporterOptions
 
         public override ExportResult Export(in Batch<Metric> batch)
         {
-            MonitoringDataPoint monitoringDataPoint = new MonitoringDataPoint();
             DocumentBuffer filledBuffer = _doubleBuffer.FlipDocumentBuffers();
+            MonitoringDataPoint monitoringDataPoint = new()
+            {
+                Version = SdkVersionUtils.s_sdkVersion.Truncate(SchemaConstants.Tags_AiInternalSdkVersion_MaxLength),
+                InvariantVersion = CurrentInvariantVersion,
+                Instance = MetricResource?.RoleInstance,
+                RoleName = MetricResource?.RoleName,
+                MachineName = Environment.MachineName,
+                StreamId = GetStreamId(),
+                Timestamp = DateTime.UtcNow,
+                //TODO: Provide feedback to service team to get this removed, it not a part of AI SDK.
+                // TransmissionTime = DateTime.UtcNow,
+                IsWebApp = IsWebAppRunningInAzure(),
+                PerformanceCollectionSupported = false,
+                // AI SDK relies on PerformanceCounter to collect CPU and Memory metrics.
+                // Follow up with service team to get this removed for OTEL based live metrics.
+                // TopCpuProcesses = null,
+                // TODO: Configuration errors are thrown when filter is applied.
+                // CollectionConfigurationErrors = null,
+            };
 
             foreach (var item in filledBuffer.ReadAllAndClear())
             {
                 monitoringDataPoint.Documents.Add(item);
             }
 
+            foreach (var metric in batch)
+            {
+                foreach (ref readonly var metricPoint in metric.GetMetricPoints())
+                {
+                    LiveMetricsExtractionProcessor.s_liveMetricNameMapping.TryGetValue(metric.Name, out var metricName);
+                    var liveMetricPoint = new Models.MetricPoint
+                    {
+                        Name = metricName
+                    };
```

Comment:
Is the TryGet necessary here?
I don't see any checks if the TryGet was successful.


One optimization, I think the TryGet could be moved out of the second foreach.